Makefile

```diff
@@ -24,4 +24,10 @@ deps: trash
 
 .DEFAULT_GOAL := ci
 
-.PHONY: $(TARGETS)
\ No newline at end of file
+.PHONY: $(TARGETS)
+
+.PHONY:
+
+.PHONY: publish-linux-amd64
+publish-linux-amd64:
+	./scripts/publish-linux-amd64
```

Comment:
We do not need this file

Reply:
fixed, added the push command directly to the makefile